positron/comms/data_explorer-backend-openrpc.json

```diff
@@ -1003,63 +1036,88 @@
 				"required": [
-					"bin_sizes",
-					"bin_width"
+					"num_bins"
 				],
 				"properties": {
-					"bin_sizes": {
+					"num_bins": {
+						"type": "integer",
+						"description": "Number of bins in the computed histogram"
+					},
+					"quantiles": {
 						"type": "array",
-						"description": "Absolute count of values in each histogram bin",
+						"description": "Sample quantiles (numbers between 0 and 1) to compute along with the histogram",
 						"items": {
-							"type": "integer"
+							"type": "number"
 						}
-					},
-					"bin_width": {
-						"type": "number",
-						"description": "Absolute floating-point width of a histogram bin"
 					}
 				}
 			},
-			"column_frequency_table": {
+			"column_histogram": {
 				"type": "object",
-				"description": "Result from a frequency_table profile request",
+				"description": "Result from a histogram profile request",
 				"required": [
-					"counts",
-					"other_count"
+					"bin_edges",
+					"bin_counts"
 				],
 				"properties": {
-					"counts": {
+					"bin_edges": {
 						"type": "array",
-						"description": "Counts of distinct values in column",
+						"description": "String-formatted versions of the bin edges, there are N + 1 where N is the number of bins",
 						"items": {
-							"$ref": "#/components/schemas/column_frequency_table_item"
+							"type": "string"
 						}
 					},
-					"other_count": {
+					"bin_counts": {
+						"type": "array",
+						"description": "Absolute count of values in each histogram bin",
+						"items": {
+							"type": "integer"
+						}
+					}
+				}
+			},
+			"column_frequency_table_params": {
+				"type": "object",
+				"description": "Parameters for a frequency_table profile request",
+				"required": [
+					"limit"
+				],
+				"properties": {
+					"limit": {
 						"type": "integer",
-						"description": "Number of other values not accounted for in counts. May be 0"
+						"description": "Number of most frequently-occurring values to return. The K in TopK"
 					}
 				}
 			},
-			"column_frequency_table_item": {
+			"column_frequency_table": {
 				"type": "object",
-				"description": "Entry in a column's frequency table",
+				"description": "Result from a frequency_table profile request",
 				"required": [
-					"value",
-					"count"
+					"values",
+					"counts"
 				],
 				"properties": {
-					"value": {
-						"type": "string",
-						"description": "Stringified value"
+					"values": {
+						"type": "array",
+						"description": "The formatted top values",
+						"items": {
+							"type": "string"
+						}
+					},
+					"counts": {
+						"type": "array",
+						"description": "Counts of top values",
+						"items": {
+							"type": "integer"
+						}
 					},
-					"count": {
+					"other_count": {
 						"type": "integer",
-						"description": "Number of occurrences of value"
+						"description": "Number of other values not accounted for in counts. May be omitted"
```

Comment:
Maybe we want to document that this excludes `NAs`/ `Nulls`? To be consistent with the numeric histograms?

Reply:
done